pensjonsmaler/src/main/kotlin/no/nav/pensjon/brev/maler/redigerbar/InnhentingInformasjonFraBruker.kt

```diff
@@ -0,0 +1,671 @@
+import no.nav.pensjon.brev.maler.fraser.common.Constants.NAV_URL
+import no.nav.pensjon.brev.maler.fraser.common.Felles
+import no.nav.pensjon.brev.template.Language.*
+import no.nav.pensjon.brev.template.LetterTemplate
+import no.nav.pensjon.brev.template.RedigerbarTemplate
+import no.nav.pensjon.brev.template.dsl.createTemplate
+import no.nav.pensjon.brev.template.dsl.expression.expr
+import no.nav.pensjon.brev.template.dsl.expression.plus
+import no.nav.pensjon.brev.template.dsl.helpers.TemplateModelHelpers
+import no.nav.pensjon.brev.template.dsl.languages
+import no.nav.pensjon.brev.template.dsl.text
+import no.nav.pensjon.brev.template.dsl.textExpr
+import no.nav.pensjon.brevbaker.api.model.LetterMetadata
+
+@TemplateModelHelpers
+object InnhentingInformasjonFraBruker : RedigerbarTemplate<InnhentingInformasjonFraBrukerDto> {
+    override val kategori: TemplateDescription.Brevkategori = TemplateDescription.Brevkategori.INNHENTE_OPPLYSNINGER
+    override val brevkontekst: TemplateDescription.Brevkontekst = TemplateDescription.Brevkontekst.ALLE
+    override val sakstyper: Set<Sakstype> = setOf(Sakstype.ALDER)
+    override val kode = Pesysbrevkoder.Redigerbar.PE_AP_INNHENTING_INFORMASJON_FRA_BRUKER     //MF_000133
+    override val template: LetterTemplate<*, InnhentingInformasjonFraBrukerDto> = createTemplate(
+        name = kode.name,
+        letterDataType = InnhentingInformasjonFraBrukerDto::class,
+        languages = languages(Bokmal, Nynorsk, English),
+        letterMetadata = LetterMetadata(
+            displayTitle = "Innhenting av opplysninger/dokumentasjon",
+            isSensitiv = false,
+            distribusjonstype = LetterMetadata.Distribusjonstype.VIKTIG,
+            brevtype = LetterMetadata.Brevtype.INFORMASJONSBREV
+        ),
+    ) {
+        title {
+            textExpr(
+                Bokmal to "Vi har mottatt ".expr() +
+                        fritekst("søknad/blankett/brev/henvendelse")
+                        + " fra deg " + fritekst("dato") + ".",
+
+                Nynorsk to "Vi har fått ".expr() +
+                        fritekst("søknad/blankett/brev/henvendelse")
+                        + " frå deg " + fritekst("dato") + ".",
+
+                English to "We have received ".expr() +
+                        fritekst("an application/a form/a letter/an inquiry")
+                        + " from you " + fritekst("dato") + ".",
+            )
+        }
+        outline {
+            paragraph {
+                val fritekst = fritekst("Utfyllende informasjon til bruker om årsak til brevet.")
+                textExpr(
+                    Bokmal to fritekst,
+                    Nynorsk to fritekst,
+                    English to fritekst,
+                )
+            }
+            paragraph {
+                text(
+                    Bokmal to "For at vi skal kunne behandle saken din ferdig, må du sende oss følgende opplysninger/ dokumentasjon:",
```

Comment:
`fritekst("opplysninger/dokumentasjon")`?
Det er også forskjell på mellomrom rundt slash for alle språklag.

Reply:
Det har blitt brukt litt varierende i maler. I denne malen skal man beholde en av verdiene, og noen ganger fylle inn ett felt (f.eks dato). Hadde løsningen vært å kunne ha en multiple choice type løsning eller lignende på det? Så du kunne ha valgt hvilken variant du vil ha. I dag må saksbehandler viske ut det de ikke skal ha.

Reply:
Ut fra engelsk språklaget ser det ut som teksten faktisk skal være generell: "information and/or documentation"